eng/Versions.props

```diff
@@ -171,6 +175,7 @@
     <VersionToolsVersion>2.2.0-beta.19072.10</VersionToolsVersion>
     <DotnetDebToolVersion>2.0.0</DotnetDebToolVersion>
     <MicrosoftNETTestSdkVersion>17.4.0-preview-20220813-01</MicrosoftNETTestSdkVersion>
+    <MicrosoftExtensionsLoggingVersion>7.0.0-preview.7.22375.6</MicrosoftExtensionsLoggingVersion>
```

Comment:
This will also need a version.details.xml element so that it doesn't hang at p7 forever.